{
  "modified": "2025-02-03T17:08:02Z",
  "published": "2025-02-03T17:08:02Z",
  "schema_version": "1.5.0",
  "id": "MAL-2025-1001",
  "summary": "Malicious code in utils-hex (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "utils-hex",
        "purl": "pkg:pypi/utils-hex"
      },
      "versions": [
        "1.0.0"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2025-00542",
        "import_time": "2025-02-03T18:38:10.202641859Z",
        "modified_time": "2025-02-03T17:08:02Z",
        "sha256": "0c27ce846a1e77d6b0b3b1058e089b829b9ecd85c0b0810880a4c81e66504235",
        "source": "reversing-labs",
        "versions": [
          "1.0.0"
        ]
      }
    ]
  }
}
